src/llmq/dkgsessionhandler.cpp

```diff
@@ -367,24 +367,22 @@ std::set<NodeId> BatchVerifyMessageSigs(CDKGSession& session, const std::vector<
     pubKeys.reserve(messages.size());
     messageHashes.reserve(messages.size());
     bool first = true;
-    for (const auto& p : messages ) {
-        const auto& msg = *p.second;
-
-        auto member = session.GetMember(msg.proTxHash);
+    for (const auto& [nodeId, msg] : messages ) {
```

Comment:
clang format complains:
```suggestion
    for (const auto& [nodeId, msg] : messages) {
```

Reply:
done